index.bs

```diff
@@ -1374,6 +1374,10 @@ ignored or an exception is thrown.
     };
 </pre>
 
+The type of an attribute that is not [=read only=] must not be a [=promise type=]. Additionally,
+the type of an attribute with any of the [=extended attributes=] [{{LenientThis}}],
+[{{PutForwards}}], [{{Replaceable}}], or [{{SameObject}}] must not be a [=promise type=].
```

Comment:
I thought there were requirements for [{{SameObject}}] promise type attributes. E.g. for https://drafts.csswg.org/css-font-loading/#font-face-set-ready, no?